tools/docker-notifications-override.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
```

Comment:
This is a neat tool for development, but I don't think it's worth maintaining another compose file and make target.  Let's remove this file and change https://github.com/ansible/awx/pull/4124/files#diff-b67911656ef5d18c4ae36cb6741b7965R568

...so that instead does:

```
docker run --network="tools_default" --name httpbin -p 8204:80 kennethreitz/httpbin
```

Reply:
For brevity/ease of use, it might also be nice to change the make target name from `make docker-compose-notifications` to `make httpbin`.

Reply:
I like the idea of renaming this to `make docker-compose-httpbin` :+1: 

This is running `docker-compose up` which isn't _quite_ the same thing as `docker run`. Using the compose file and `up` for the make target means starting a new container or attaching to an existing one as needed depending on context.

I'm not sure what you mean by ease of use since they'd both use the same make target.

Another benefit here is consistency - this is how we make similar extensions to the development environment for other services.

Reply:
@jakemcdermott 

My thinking was that there's a pretty small set of situations where you want this functionality and my thought was "I'm debugging notifications and it'd be nice to just quickly spin up a local HTTP service I can point at".

An easy alias for `docker run --network="tools_default" --name httpbin -p 8204:80 kennethreitz/httpbin` that runs in the foreground and attaches to the default network in an -_additive_ way would give you what you need.  We _have_ done this in other places before to test notification services:

https://github.com/ansible/awx/blame/devel/docs/notification_system.md#L70
https://github.com/ansible/awx/blame/devel/docs/notification_system.md#L107

Reply:
> One of my concerns is that is we do it this way, we'll end up growing an extra docker compose override file per service we'd potentially want to test.

I actually don't see the downside here - override files for additional services and extensions is using docker-compose the way the documentation instructs.

That said, you're right that we already have some `run` examples in the docs so why not - that wfm.

Reply:
It should also be said that I don't have _very_ strong feelings about this, and I'm not interesting in holding off this (otherwise solid) PR due to it.  If we want to change it up some, @bianca let's do it after this merges.